compiler/src/main/scala/dx/compiler/ApplicationCompiler.scala

```diff
@@ -166,7 +167,14 @@ case class ApplicationCompiler(typeAliases: Map[String, Type],
         instanceTypeDb.apply(static.toInstanceTypeRequest).name
       case DefaultInstanceType | DynamicInstanceType =>
         // TODO: should we use the project default here rather than
```

Comment:
What is meant by this -- I can resolve this todo in the PR if you want? @jdidion